src/components/multi_select_state.js

```diff
@@ -130,11 +130,15 @@ const withMultiSelectState = WrappedComponent =>
     }
 
     filterItems(event) {
-      const { items, filterFunction } = this.props;
+      const { items, filterFunction, searchValueChanged } = this.props;
       const { value } = event.target;
       this.setState({
         filteredItems: items.filter(filterFunction(value))
       });
+
+      if (typeof searchValueChanged === "function") {
```

Comment:
Please check that it's not empty instead of function
Something like: 
`searchValueChanged && searchValueChanged(value);`

Reply:
Fixed